tests/codemod/deprecate-version/transformer.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import { execSync } from "child_process";
+import * as fs from "fs";
+import * as path from "path";
+import { expectedOutputCode, inputCode } from "./test-data";
+
+describe("Codemod Transformer for updating version", () => {
+    const inputPath = path.join(__dirname, "input.tsx");
+    const outputPath = path.join(__dirname, "output.tsx");
+
+    beforeAll(() => {
+        // create sample input file for testing
+        jest.resetAllMocks();
+        fs.writeFileSync(inputPath, inputCode);
+        console.log(`Input file created at: ${inputPath}`);
+        console.log(`Input code:\n${inputCode}`);
+    });
+
+    afterAll(() => {
+        // To delete the files created for testing comment to view the files
+        fs.unlinkSync(inputPath);
+        fs.unlinkSync(outputPath);
+    });
+
+    it("should transform the code to updated version", () => {
+        // copy the input file to the transformed output file
+        fs.copyFileSync(inputPath, outputPath);
+        console.log(`Output file created at: ${outputPath}`);
+
+        // do the jscodeshift command
+        console.log(`Running jscodeshift command...`);
+        execSync(
+            `jscodeshift -t ../react-design-system/codemods/deprecate-version ${outputPath}`
```

Comment:
no need to navigate to the parent folder as jest is already running in the project directory

```suggestion
            `jscodeshift -t ./codemods/deprecate-version ${outputPath}`
```